tables/tr-g2.ctb

```diff
@@ -0,0 +1,599 @@
+﻿# liblouis: Türkçe Tam Kısaltmalı Braille
+#
+#  Copyright (C) 2009 SBS Schweizerische Bibliothek für Blinde und Sehbehinderte

```

Comment:
I'm not the table author, I'm merely tasked with integrating it into liblouis :)